src/dsp_tools/utils/stack_handling.py

```diff
@@ -35,9 +35,9 @@ def start_stack(
     if enforce_docker_system_prune and suppress_docker_system_prune:
         raise BaseError('The arguments "--prune" and "--no-prune" are mutually exclusive')
 
-    # copy contents of knora/dsplib/docker to ~/.dsp-tools/docker
+    # copy contents of src/dsp_tools/docker to ~/.dsp-tools/docker
     # rationale to use importlib.resources: https://setuptools.pypa.io/en/latest/userguide/datafiles.html#accessing-data-files-at-runtime
```

Comment:
is this still up to date?

Reply:
Yes, the contents must be copied to `~`, that's up to date.
But no, using `importlib.resources` is not the most up to date way how to do this. According to https://stackoverflow.com/a/58941536/14414188, I should switch to https://docs.python.org/3.9/library/pkgutil.html.
There are 2-3 places in the entire code base where I have to add pkgutil, that is on my TODO list.